webtool/gradle/common/src/commonMain/kotlin/Pilot.kt

```diff
@@ -68,6 +68,27 @@ data class Pilot(
         Json.encodeToString(this)
 }
 
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+fun pilotFromJson(json: String) = Json.decodeFromString<Pilot>(json)
+
+@OptIn(ExperimentalJsExport::class)
+@JsExport
+fun createDefaultStartPilot() = Pilot(
+    name = "start",
+    householdGroups = listOf(
```

Comment:
I think it works without an initial setup, it's been fixed and the simulation side. Check with @GillisHommen

Reply:
It works without households. @GillisHommen do you want the initial households removed?